src/components/survey.js

```diff
@@ -14,12 +14,14 @@ module.exports = function(app) {
           angular.forEach(component.values, function(v) {
             values[v.value] = v.label;
           });
-          angular.forEach(component.questions, function(question) {
-            view += '<tr>';
-            view += '<th>' + question.label + '</th>';
-            view += '<td>' + values[data[question.value]] + '</td>';
-            view += '</tr>';
-          });
+		  if(data) {
```

Comment:
This happens because you use tabs. Please configure your IDE to use spaces and this will line up the way it should.